process-engine/process_engine_runtime
# ✨ Add a Route to Query the used Authority

**Changes:**

1. Adds an `/security/authority` route which returns the authority used by the runtime.
2. Introduces a Configuration file where every Global Route can be defined.
3. Add integrationtests for the new routes
4. Refactor the global config initialization:
    - When providing a value to `CONFIG_PATH` at startup, that value will no longer be overwritten by the default `config` folder.
    - Providing absolute paths to `CONFIG_PATH`, that path will be used instead of the default path.
    - It is now possible to use environments other than `sqlite` or `postgres`. Those environments must be provided by the `NODE_ENV` variable.
    - Checking for the existence of a CONFIG_PATH is now done before any other step. If the check fails, the runtime will be terminated with an error.

**Issues:**

Closes #210 

PR: #213 

## How can others test the changes?

- Start the runtime
- Navgate to `http://localhost:8000/security/authority` and see if the correct url is returned.
- Attempt to use any mix of `NODE_ENV` and `CONFIG_PATH` to test for various configuration scenarios.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
